bot/cogs/owner_cog.py

```diff
@@ -165,13 +165,13 @@ async def get(self, ctx, lines: int):
     @eval_bot.command()
     @commands.is_owner()
     async def bot(self, ctx, *, code):
-        code = code.strip('`')
-        res = eval(code)
-        if inspect.isawaitable(res):
-            res = await res
-
-        
-        await ctx.send(res)
+        code = code.replace('```python', '')
+        code = code.replace('`', '')
+        code = code.replace('\n', '\n\t')
```

Comment:
how does this handle spaces?

Reply:
It can handle spaces as indents as long as its 5 spaces